docs/contracts.rst

```diff
@@ -115,7 +115,7 @@ This means that cyclic creation dependencies are impossible.
         {
             // Check some arbitrary condition.
             address tokenAddress = msg.sender;
-            return (keccak256(abi.encodePacked(newOwner)) & 0xff) == (bytes20(tokenAddress) & 0xff);
+            return (uint256(keccak256(abi.encodePacked(newOwner))) & 0xff) == (uint160(bytes20(tokenAddress)) & 0xff);
```

Comment:
This is just a random check, we can replace it by something else. Especially this early in the documentation, we should not have complicated code. Can you find something else?